azurelinuxagent/common/osutil/alpine.py

```diff
@@ -34,7 +34,7 @@ def is_dhcp_enabled(self):
     def get_dhcp_pid(self):
         return self._get_dhcp_pid(["pidof", "dhcpcd"])
 
-    def restart_if(self, ifname): # pylint: disable=W0221
+    def restart_if(self, ifname, unused_retries=None, unused_wait=None):
```

Comment:
we should keep the parameter names consistent, too, in case somebody ever does "restart_if(..., retries=...)